### ISSUE TITLE
Allow streaming/interactive /run commands

### ISSUE THREAD

**Issue by paul-gauthier** [paul-gauthier/aider#834]

**Comment by DanaelP** [comment]
I will be interesting in this feature. Currently, 

I've done the following change in run_cmd.py function run_cmd_pexpect:
`child = pexpect.spawn(shell, args=["-i", "-c", command], encoding="utf-8")`

**Comment by paul-gauthier** [comment]
I'm going to close this issue for now, but feel free to add a comment here and I will re-open. Or feel free to file a new issue any time.

**Comment by DanaelP** [comment]
You are right. The change I mentioned allow to have the command send to an interactive shell where ~/.bashrc is sourced so the alias set there are available from /run directly.
